curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64649f91446721e3ff391d88.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-97
 
 # --description--
 
-Finally, use string interpolation to set the text of the paragraph element to `decimalToBinary(${currVal})`, where `currVal` is the `inputVal` property of the current object. After this, test out your code by entering the number `5` into the number input and clicking the `Convert` button.
+Finally, use string interpolation to set the text of the paragraph element to `"decimalToBinary(${currVal})"`, where `currVal` is the `inputVal` property of the current object. After this, test out your code by entering the number `5` into the number input and clicking the `Convert` button.
 
 # --hints--
 
-You should use string interpolation to set the text of the paragraph element to `decimalToBinary(${obj.inputVal})`.
+You should use string interpolation to set the text of the paragraph element to `"decimalToBinary(${obj.inputVal})"`.
```

Comment:
campers are asked to use string interpolation to set the text to `decimalToBinary(${obj.inputVal})` wrapping this string in quotes will also cause further confusion

Reply:
Same as above

Reply:
campers are asked to set the text of the paragraph element to `decimalToBinary(${currVal})` where `currVal` is the `inputVal` property of the current object.....  something like the following 
```js
<p id="${obj.inputVal}" style="margin-top: ${obj.marginTop}px;" class="animation-frame"> decimalToBinary(${obj.inputVal})
 </p>
```
wrapping the `"decimalToBinary(${currVal})"`  would result in the following 
```js
<p id="${obj.inputVal}" style="margin-top: ${obj.marginTop}px;" class="animation-frame"> " decimalToBinary(${obj.inputVal}) "
 </p>
```
which in this case would not pass the test

Reply:
Thanks @RGHANILOO , addressed